code/src/java/pcgen/cdom/helper/Aspect.java

```diff
@@ -107,7 +107,7 @@ public Aspect(final AspectName key, final String aString)
 	private void parseAspectString(final String aString)
 	{
 		int currentInd = 0;
-		int percentInd = -1;
+		int percentInd;
```

Comment:
Not sure this was useful. You've removed a qualifier value in place to prevent something... Was this tested?

Reply:
It ran through the automated tests fully, although I didn't know they weren't considered complete. percentInd is immediately reassigned by the condition on the while loop, so this -1 should never make any difference.